utils/ANLEra_help.cfg

```diff
@@ -147,6 +157,13 @@
                                 {ANL_HELP_FARMS}
                             [/command]
                         [/option]
+						
```

Comment:
This is currently in the textdomain for strings reused from mainline (see line 154).
And needs to be switched back afterwards to let the "villages" (line 169) remain in the mainline textdomain.

For textdomains, they do not follow macros, it counts only in what domain the strings are in the source code.
The other new text needs as well to be in the ANLEra textdomain to be translatable.